### PR TITLE
Added -notex option to allow skipping texture output

### PR DESCRIPTION
**Pull request by ventorvar**
Addresses #83

### CODE REVIEW

File: CgfConverter/Renderers/Collada/COLLADA.cs [hunk 1]
```diff
@@ -55,7 +55,8 @@ public void GenerateDaeObject()
             
             WriteRootNode(colladaVersion);
             WriteAsset();
-            WriteLibrary_Images();
+            if (! Args.NoTextures)
```
Comment (Markemp) [inline CgfConverter/Renderers/Collada/COLLADA.cs:58]: Please remove space between ! and Args.

File: CgfConverter/Renderers/Wavefront/Wavefront.Material.cs [hunk 2]
```diff
@@ -65,73 +65,77 @@ public void WriteMaterial(CryEngine cryEngine)
...
+                                    break;
+
+                                case CryEngineCore.Material.Texture.MapTypeEnum.Specular:
+                                    file.WriteLine("map_Ks {0}", textureFile.ToString());
+                                    file.WriteLine("map_Ns {0}", textureFile.ToString());
+                                    break;
+
+                                case CryEngineCore.Material.Texture.MapTypeEnum.Bumpmap:
+                                case CryEngineCore.Material.Texture.MapTypeEnum.Detail:
+                                    // <Texture Map="Detail" File="textures/unified_detail/metal/metal_scratches_a_detail.tif" />
```
Comment (Markemp) [inline CgfConverter/Renderers/Wavefront/Wavefront.Material.cs:96]: Can you remove unused comments?  I know they are left over from when I originally wrote them (bad, I know!), but as long as this file is being modified, might as well clean it up. :)

File: CgfConverter/Renderers/Wavefront/Wavefront.Material.cs [hunk 3]
```diff
@@ -65,73 +65,77 @@ public void WriteMaterial(CryEngine cryEngine)
...
+
+                                case CryEngineCore.Material.Texture.MapTypeEnum.SubSurface:
+                                    // <Texture Map="SubSurface" File="objects/spaceships/ships/aegs/textures/interior/atlas/aegs_int_atlas_retaliator_spec.tif"/>
+                                    file.WriteLine("map_Ns {0}", textureFile.ToString());
+                                    break;
+
+                                case CryEngineCore.Material.Texture.MapTypeEnum.Custom:
+                                    // <Texture Map="Custom" File="objects/spaceships/ships/aegs/textures/interior/metal/aegs_int_metal_painted_red_ddna.tif"/>
+                                    // file.WriteLine("decal {0}", textureFile);
+                                    break;
```
Comment (Markemp) [inline CgfConverter/Renderers/Wavefront/Wavefront.Material.cs:118]: For cases that don't do anything, can you collapse them all together with the default case?

```
case CryEngineCore.Material.Texture.MapTypeEnum.Custom:
case CryEngineCore.Material.Texture.MapTypeEnum.BlendDetail:
case CryEngineCore.Material.Texture.MapTypeEnum.Environment:
default:
```

I think that works...

**Review by Markemp — CHANGES_REQUESTED** [review]
Looks good.  Just some simple cleanup please.

**Comment by ventorvar** [comment]
Happy to!  Cleaned up and I did test the `obj` output after cleanup. Works the same as it did before 👍  And apologies if it looks like I swung in and picked this up, didnt realize you had assigned it to yourself and i was already working on when vmxeo put in the ticket >D

**Review by Markemp — APPROVED** [review]
Thank you for the updates!

**Pull request merged**